src/main/java/com/nekomaster1000/infernalexp/items/IWhipItem.java

```diff
@@ -0,0 +1,19 @@
+package com.nekomaster1000.infernalexp.items;
+
+import java.util.UUID;
+
+public interface IWhipItem {
+	UUID REACH_MODIFIER_UUID = UUID.fromString("c6aac07c-3300-4994-be4a-8d42d171de2e");
```

Comment:
Should be in a util class imo

Reply:
I like keeping it to the whip for now since it makes it easy to find and it's also the only thing that uses it. If we use it more in the future, I could see moving it to a util class, but currently being specific to the whip, I think it's fine where it is.